code/src/components/Details.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react'
+import { NavLink, useParams } from 'react-router-dom'
+
+import { BackIcon } from './BackIcon'
+
+const Details = () => {
+
+    const [details, setDetails] = useState([])
+    const { id } = useParams()
+
+    useEffect(() => {
+        fetch(`https://api.themoviedb.org/3/movie/${id}?api_key=${'2ef0cecd459c1c1851bbe1d699a604a0'}&language=en-US`)
+            .then(res => res.json())
+            .then(data => {
+                console.log(data)
+                setDetails(data)
+            })
+        }, [id])
+
+    return (
+            <div className="details-body"> 
+                <img className="movie-poster" src={`https://image.tmdb.org/t/p/w1280${details.backdrop_path}`} alt={details.title}/>
+                
+                <NavLink class="backLink" to='/'>
+                   <BackIcon />
+                </NavLink>
+                
+                <div className="movie-box">
+                    <img className="movie-image" src={`https://image.tmdb.org/t/p/w342${details.poster_path}`} alt={details.title} tabindex="0"/>
```

Comment:
I like that you thought of keyboard accessibility by using the tabindex attribute, tried it and it works :)